apps/mull-ui/src/app/pages/messages/event-messages/event-message-list.tsx

```diff
@@ -24,11 +26,12 @@ export const EventMessageList = () => {
         eventPicture={mediaUrl(event)}
         eventDate={new Date(event.startDate)}
         key={'event-bullet-' + index}
+        onClick={() => history.push(`${ROUTES.ANNOUNCEMENTS.url.replace(':id', `${event.id}`)}`)}
```

Comment:
Something funky is going on with the onClick as it will stick to the first event I clicked on and won't change afterwards.
For example, if I click on event 1, any other event that I click will lead me to event 1's chat.
The only way I can get this to change is to manually change the id in the URL, however it will get stuck to that id once again.

Reply:
I'm getting this too. Debugged it a little and found that the event.id value is correct. Might be some issue with our routes.

Reply:
This is still happening for me :s